pkgs/development/ocaml-modules/zmq/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, ocaml, findlib, dune, czmq, stdint }:
+
+stdenv.mkDerivation rec {
+  name = "ocaml${ocaml.version}-zmq-${version}";
+  version = "20180726";
+  src = fetchFromGitHub {
+    owner = "issuu";
+    repo = "ocaml-zmq";
+    rev = "d312a8458d6b688f75470248f11875fbbfa5bb1a";
+    sha256 = "1f5l4bw78y4drabhyvmpj3z8k30bill33ca7bzhr02m55yf6gqpf";
+  };
+
+  patches = [
+    ./ocaml-zmq-issue43.patch
+  ];
+
+  buildInputs = [ ocaml findlib dune stdint czmq ];
+
+  buildPhase = "dune build -p zmq";
+
+  inherit (dune) installPhase;
+
+  meta = {
+    description = "ZeroMQ bindings for OCaml";
+    license = stdenv.lib.licenses.mit;
+    inherit (src.meta) homepage;
```

Comment:
please add a `platforms` attribute

Reply:
Is it OK to just do the inherit here for that?